Content Updates (#143)
bitwarden/contributing-docs: docs/getting-started/server/guide.md

## Install Licensing Certificate

To run your local server environment as a licensed instance:

1. Go to Bitwarden's shared Development collection. (If you don't have access to this collection,
   contact our IT manager. Make sure you have first set up a Bitwarden account using your company
   email address.)
2. Search for the `Licensing Certificate - Dev` credentials.

> 3. Download the `dev.pfx` certificate and double-click to install it. You will be prompted to enter

Author: I used the `dev.pfx` file, but not sure if the `dev.cer` file would work fine as well?

Reviewer: I'm not sure. To be honest I don't even know what the licensing certificate does.

## Install Licensing Certificate

To run your local server environment as a licensed instance:

> 1. Go to Bitwarden's shared Development collection. (If you don't have access to this collection,

Author: The Engineering collection is now called Development. As for the note about contacting IT if necessary, I've moved this note here because this is the first place in the setup guide that mentions the Development Collection.

Reviewer: Makes sense!

We provide a helper script which simplifies setting user secrets for all projects in the server
repository.

> 1.  Get a `secrets.json` template so you can modify it with your own secrets values.

Reviewer: I think this can be simplified further:

```suggestion
1.  Create a `secrets.json` file from the template:
```

    <bitwarden>

>     1. Go to Bitwarden's shared Development collection.
>     2. Search for the `Server User Secrets` credentials.
>     3. Download the `secrets.json` file and copy it into the server `dev` folder. (This
>        `secrets.json` file is configured to use the dockerized Azurite and MailCatcher instances and
>        is recommended for this guide.)

Reviewer: No full stops are needed for numbered lists. (It's probably needed for the last item because of the separate sentence/parens, but that's an exception)

```suggestion
    1. Go to Bitwarden's shared Development collection
    2. Search for the `Server User Secrets` credentials
    3. Download the `secrets.json` file and copy it into the server `dev` folder. (This
       `secrets.json` file is configured to use the dockerized Azurite and MailCatcher instances and
       is recommended for this guide.)
```
